cnb/cnb_lifecycle.go

```diff
@@ -0,0 +1,67 @@
+//go:build !noInternet && !noCNB
+// +build !noInternet,!noCNB
```

Comment:
Why add these build tags? I did a search and see that there are a couple similar build tags, but I don't see them justified anywhere. Seems like we should lean into using our configuration parameters to skip tests we don't want to run.

Reply:
We started with a copy of the docker lifecycle tests. If those tags are not used/mandatory, we can just remove them.

Reply:
Ok, AFAIK they're unused. If you could remove them please, that would be great.

Reply:
Oh seems like you did, thanks.